$OpenBSD: patch-panels_common_gdm-languages_c,v 1.2 2011/09/27 12:15:23 ajacoutot Exp $

OpenBSD does not provide localedef(1), hence we don't have a
locale-archive file.

--- panels/common/gdm-languages.c.orig	Tue Aug 23 10:23:42 2011
+++ panels/common/gdm-languages.c	Mon Sep 26 14:01:06 2011
@@ -455,6 +455,7 @@ struct nameent
 static gboolean
 collect_locales_from_archive (void)
 {
+#ifndef __OpenBSD__
         GMappedFile        *mapped;
         GError             *error;
         char               *addr;
@@ -512,6 +513,9 @@ collect_locales_from_archive (void)
 
         g_mapped_file_unref (mapped);
         return locales_collected;
+#else
+        return FALSE;
+#endif
 }
 
 static int
